lib/MySQL_Session.cpp

```diff
@@ -5768,16 +5784,18 @@ bool MySQL_Session::handler___status_WAITING_CLIENT_DATA___STATE_SLEEP___MYSQL_C
 						proxy_debug(PROXY_DEBUG_MYSQL_COM, 5, "Processing SET %s value %s\n", var.c_str(), value1.c_str());
 #endif // DEBUG
 					} else {
-						// if we reach here, proxysql didn't recognize every variable in a set statement
```

Comment:
@renecannao this is probably one of the most relevant changes to take into account.